src/locale/pt-br.ts

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable @typescript-eslint/camelcase */
+/* spellchecker: disable */
+
+const translations = {
+  actions: {
+    new: 'Criar novo(a)',
```

Comment:
Brazilian Portuguese locale is already merged, why do you have this in your PR? 🤔